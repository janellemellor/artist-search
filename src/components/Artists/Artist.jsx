import React from 'react';
import PropTypes from 'prop-types';

const Artist = ({ name }) => (
  <>
    <p>{name}</p>
  </>

);

Artist.propTypes = {
  name: PropTypes.string.isRequired
};

export default Artist;

