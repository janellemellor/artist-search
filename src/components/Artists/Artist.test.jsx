import React from 'react';
import { shallow } from 'enzyme';
import Artist from './Artist.jsx';

describe('Artist Component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Artist name ="U2" />);
    expect(wrapper).toMatchSnapshot();
  });
});
