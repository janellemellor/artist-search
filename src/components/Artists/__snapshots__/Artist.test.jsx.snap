// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist Component matches a snapshot 1`] = `
<Link
  to="/U2/undefined"
>
  <p>
    U2
  </p>
</Link>
`;
